src/MonitorProfiler/MainProfiler/MainProfiler.cpp

```diff
@@ -91,3 +97,34 @@ HRESULT MainProfiler::InitializeLogging()
 
     return S_OK;
 }
+
+HRESULT MainProfiler::InitializeCommandServer()
+{
+    HRESULT hr = S_OK;
+
+    tstring instanceId;
+    IfFailRet(m_pEnvironment->GetEnvironmentVariable(_T("DotnetMonitorProfiler_InstanceId"), instanceId));
```

Comment:
Can you move this to EnvironmentHelper? I'd like to try to keep all of the `DotnetMonitorProfiler_` variables in one spot for easy determination of what variables are in use.